application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -172,7 +174,7 @@
   const initialiseJobJS = function() {
     require.undef('health-check-job');
     $.get(getResourceURL(), function(data) {
-      let jobMinJsURL = $(data).find('input[name="jobMinJsURL"]').val();
+      let jobMinJsURL = $(data).find('jobJsURL').attr('data-url');
```

Comment:
Is ``jobJsURL`` a "custom" HTML element? I don't see where you [define this custom element](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components/Using_custom_elements#implementing_a_custom_element) and I don't see why you need a custom element, when you can simply add a CSS class on the element you want to locale.